HTTPlayground/settings.py

```diff
@@ -1,13 +1,25 @@
 from pathlib import Path
 from sys import argv
+from jinja2 import Environment, FileSystemLoader
 
-PORT = 8000
-URL = 'localhost'
+SERVERS_INFO = {
+    'URL0': '127.0.0.1', 'PORT0': 8000,
+    'URL1': '0.0.0.0', 'PORT1': 8080  # 127.255.255.254
+}
 
-if len(argv) > 2:
-    PORT = int(argv[2])
-    URL = argv[1]
+IS_MAIN_SERVER = not int(argv[1])
```

Comment:
minor: this doesn't look explicit at all. Always try to think of more self-explanatory implementation options.

Reply:
advice: define smth like `main|accessory` values for the argv, not ints. They would speak more to the user (who may be a different person in theory)